src/ui-kit/experimental/patterns/layout/components/expansion/expansion-panel.html

```diff
@@ -1,8 +1,8 @@
 <ng-content select="mat-expansion-panel-header, md-expansion-panel-header, sam-expansion-panel-header"></ng-content>
-<div [class.mat-expanded]="expanded" class="mat-expansion-panel-content"
+<div [attr.aria-hidden]="!expanded" [class.mat-expanded]="expanded" class="mat-expansion-panel-content"
```

Comment:
This change completely negates the existence of [class.mat-expanded]="expanded". I do not think this looks correct but you could stop by and we could discuss it.

Reply:
I am WFH today. aria property is for 508 right? so with class.mat-expanded we are not adding aria label at all.

Reply:
if expanded is not true then we set the aria-hidden to true isn't it?

Reply:
Aria Hidden is a valid property but how this is built using the material properties and classes I am not sure that is the correct fix because of the design. Hence my concern.

Reply:
so when I tested it did add aria-hidden = true otherwise without this there was no aria-hidden at all